src/test/java/com/weaverplatform/importer/xmi/ImportXmiTest.java

```diff
@@ -9,47 +10,48 @@
  */
 public class ImportXmiTest {
 
-  private String argument0 = "http://192.168.99.100:9487";
-  private String argument1 = "/users/jonathansmit/Downloads/InformatieBackboneModel.xml";
-  private String argument2 = "testDataset";
+  private String weaverUrl = "http://docker:9487";
+  private String xmiPath = "/Users/bastiaan/Desktop/InformatieBackboneModel.xml";
+  private String datasetName = "model";
 
```

Comment:
Again, these need to be stored in a configuration file. See the weaver-sdk-java project for a quick and dirty way to provide a configuration file.